test/schema_test.exs

```diff
@@ -2,6 +2,33 @@ defmodule SchemaTest do
   use PowerAssert
   import Masdb.Schema
 
+  test "get_pk works when single column pk" do
+    c1 = %Masdb.Schema.Column{is_pk: true,  name: "c1", type: :int}
+    c2 = %Masdb.Schema.Column{is_pk: false, name: "c2", type: :int}
+    c3 = %Masdb.Schema.Column{is_pk: false, name: "c3", type: :int}
+
+    schema = %Masdb.Schema{name: "foo", replication_factor: 0, columns: [c1, c2, c3]}
+    assert Masdb.Schema.get_pk(schema) == ["c1"]
+  end
+
+  test "get_pk works when multi columns pk" do
+    c1 = %Masdb.Schema.Column{is_pk: true,  name: "c1", type: :int}
+    c2 = %Masdb.Schema.Column{is_pk: true, name: "c2", type: :int}
+    c3 = %Masdb.Schema.Column{is_pk: false, name: "c3", type: :int}
+
+    schema = %Masdb.Schema{name: "foo", replication_factor: 0, columns: [c1, c2, c3]}
+    assert Masdb.Schema.get_pk(schema) == ["c1", "c2"]
+  end
+
+  test "get_non_nullables returns non nullable cols" do
```

Comment:
I assumed that the Schema would be valid, but it's true that it could be invalid. I will add this test.
No, I don't think it should return an error.